R/after-wrappers.R

```diff
@@ -208,8 +208,10 @@ class(pl) = c("pl_polars_env", "environment")
 #' @return NULL
 #' @keywords functions
 #' @examples
+#' \dontrun{
 #' pl$show_all_public_functions()
-pl$show_all_public_functions = function() {
+#' }
```

Comment:
Why need this?

Reply:
Some stuff I tried but forgot to remove, thanks